### DISCUSSION TITLE
different output when using proxy

### DISCUSSION THREAD

**Discussion by mridulsg** [projectdiscovery/httpx#591] [Q&A]
when using proxy, limited number of valid urls are displayed where as removing the proxy gives more result. Below are the images for reference

Version: 1.2.0

With Proxy
![image](https://user-images.githubusercontent.com/21661734/156784085-0c1f5292-a2f2-439c-9069-208161298945.png)

Without proxy
![image](https://user-images.githubusercontent.com/21661734/156784295-3ad333d6-cdd9-4a04-8b02-0ece5f810d41.png)

You can try with a file with many javascript endpoints

**Comment by Mzack9999** [comment]
The issue doesn't seem reproducible. I get the same results with/without proxy. The only thing I can think of is that by using `-fc 200`, displaying the endpoints that received a status code different from 200, their number (failures) increases when using the proxy. The proxy running is Burp Suite? Can the internal box where the proxy runs (I assume it has a private IP) correctly communicate with the targets?
If you could provide a target that appears with the proxy and doesn't without it, along with the proxy setup, I will attempt to reproduce it.
